File: programmers/js/new/level1/소수_찾기.js
function solution(n) {
    var answer = 0;
    const check = Array.from(Array(n + 1), () => true);
    for (let i = 2; i < check.length; ++i) {
        if (check[i]) {
            answer++;
            for (let j = i; j < check.length; j += i) {
                check[j] = false;
            }
        }
    }
    return answer;
}

File: programmers/python/level1/소수_찾기.py
def solution(n):
    answer = 0
    visited = [False for i in range(n+1)]
    for num in range(2, n+1):
        if visited[num] == False:
            answer += 1
            for num2 in range(num, n+1, num):
                visited[num2] = True
    return answer